docs/2.19.x/concepts/flows.mdx

```diff
@@ -120,9 +120,11 @@ For example, you can provide a `name` value for the flow. Here we've also used t
 from prefect import flow
 from prefect.task_runners import SequentialTaskRunner
 
-@flow(name="My Flow",
-      description="My flow using SequentialTaskRunner",
-      task_runner=SequentialTaskRunner())
+@flow(
+    name="My Flow",
+    description="My flow using SequentialTaskRunner",
+    task_runner=SequentialTaskRunner())
```

Comment:
Why are we keeping the `SequentialTaskRunner` in this example? This example doesn't seem specific to the `SequentialTaskRunner`

Reply:
this is 2.19.x, its showing how to use settings